.travis.yml

```diff
@@ -1,19 +1,18 @@
-language: node_js
-sudo: false
-node_js:
-  - "6"
-  - "8"
-  - "10"
-  - "11"
-
-script:
-  - npm run compile
-  - npm run test
-
-after_success:
-  - npm run publish-coverage
-
-
-cache:
-  directories:
-    - "node_modules"
+language: node_js
+sudo: false
+node_js:
+  - "8"
+  - "10"
+  - "11"
```

Comment:
Can we swap this with 12 (which is now current LTS version)? Don't think we need to test for any non-LTS versions below current LTS version.

Reply:
```suggestion
  - "12"
```